packages/components/checkbox/src/less/checkbox.less

```diff
@@ -4,6 +4,8 @@
 
 // Checkbox - Base
 .oui-checkbox {
+  position: relative;
```

Comment:
Should be in `choice-base()`. This mixin is used as a base for the `checkbox` and `radio`.